hawk.go

```diff
@@ -5,23 +5,18 @@ import (
 	"crypto/sha256"
 	log "github.com/sirupsen/logrus"
 	"go.mozilla.org/hawk"
-	"go.mozilla.org/mozlogrus"
```

Comment:
Why are we removing this?

Reply:
This was changed in https://github.com/mozilla-services/tigerblood/pull/102/commits/6b350ec7a85ded2533549d8fb1b75c5bdf2fe96a

The same `init` function was defined in multiple files, so I removed it from all but `tigerblood.go`. This didn't appear to have any effect, but if it was like this for a reason I can revert this one.